### PR TITLE
update PR template

### PR DESCRIPTION
**Pull request by eclare108213**
Make the PR template less cryptic.  Is this too wordy?

### CODE REVIEW

**Comment by eclare108213** [comment]
@duvivier Please take a look at this.  If it makes sense to you, I think you can merge this PR.  Or I can merge it, or you can directly edit the file yourself and I can close this PR.  Either way, we should make the same changes to the template for Icepack.  Thanks!

**Comment by duvivier** [comment]
I think that the changes make sense. I agree it's better to be explicit, especially as we work out the sphinx documentation and users get used to the new workflow. I'll update the Icepack PR template.
